test/plugin-unit-tests.ts

```diff
@@ -1,13 +1,16 @@
 import { Plugin } from '../src/modules/plugin';
 import { screenshotResponse } from './resources/screenshot-response';
 import { sideloadResponse } from './resources/sideload-response';
-import nock = require('nock');
-import assert = require('assert');
+import chai = require('chai');
 import fs = require('fs');
+import nock = require('nock');
```

Comment:
should these be imports instead of require?

Reply:
Fixed in 59afaf3